mappings/net/minecraft/client/render/entity/EntityRendererEnderDragon.mapping

```diff
@@ -1,5 +1,12 @@
 CLASS none/bta net/minecraft/client/render/entity/EntityRendererEnderDragon
+	FIELD a BEAM_TEX Lnone/kp;
+	FIELD k DRAGON_EXPLOSION_TEX Lnone/kp;
+	FIELD l DRAGON_SKIN Lnone/kp;
```

Comment:
`EXPLOSION_TEX`
`SKIN`

Also what is beam, out of curiosity? The "beam" thing coming out of the explosion? Maybe call it `EXPLOSION_BEAM_TEX` ?